payment-instruments/mobile-pay/seamless-view.md

```diff
@@ -0,0 +1,305 @@
+    obtained in the `POST` request in the `<script>` element. Example:
+
+```html
+    <script id="payment-page-script" src="https://ecom.dev.payex.com/mobilepay/core/ scripts/client/px.mobilepay.client.js"></script>
+```
+
+The previous two steps gives this HTML:
+
+{:.code-view-header}
+**HTML**
+
+```html
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Swedbank Pay Seamless View is Awesome!</title>
+        <!-- Here you can specify your own javascript file -->
+        <script src=<YourJavaScriptFileHere>></script>
+    </head>
+    <body>
+        <div id="swedbank-pay-seamless-view-page">
+          <script id="payment-page-script" src="https://ecom.dev.payex.com/mobilepay/core/scripts/client/px.mobilepay.client.js"></script>
+        </div>
+    </body>
+</html>
+```
+
+Lastly, initiate the Seamless View with a JavaScript call to open the `iframe`
+embedded on your website.
+
+{:.code-view-header}
+**JavaScript**
+
+```js
+<script language="javascript">
+    payex.hostedView.mobilepay({
+        // The container specifies which id the script will look for to host the
+        // iframe component.
+        container: "swedbank-pay-seamless-view-page"
+    }).open();
+</script>
+```
+
+## MobilePay Online Seamless flow
+
+The sequence diagram below shows the two requests you have to send to
+Swedbank Pay to make a purchase.
+The links will take you directly to the API description for the specific
+request.
+
+```mermaid
+sequenceDiagram
+    activate Merchant
+    Merchant->>-SwedbankPay: POST /psp/mobilepay/payments
+    activate SwedbankPay
+    note left of Merchant: First API request
+    SwedbankPay-->>-Merchant: Payment response with rel: view-payment
+    activate Merchant
+    Merchant->>-SwedbankPay: Script init of iFrame
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: Display Payment Page
+    activate Merchant
+    Merchant->>Merchant: Init payment
+    Merchant->>-SwedbankPay: Init request
+
+    activate SwedbankPay
+    SwedbankPay->>+MobilePay_API: POST <rel:create-authorization>
+    activate MobilePay_API
+    MobilePay_API-->>+SwedbankPay: Response
+    activate SwedbankPay
+    SwedbankPay-->>-Merchant: Display instructions page
+
+    MobilePay_API-->>-MobilePay_App: Confirm Payment UI
+    activate MobilePay_App
+    MobilePay_App-->>MobilePay_App: Confirmation Dialogue
+    MobilePay_App-->>-MobilePay_App: Confirmation
+
+    activate MobilePay_API
+    MobilePay_API->>-SwedbankPay: Make Payment
+    activate SwedbankPay
+    SwedbankPay-->>-SwedbankPay: Process Payment
+    activate SwedbankPay
+    SwedbankPay-->>-MobilePay_API: Process Payment Response
+    activate MobilePay_API
+    MobilePay_API-->>-MobilePay_App: Transaction Status
+
+    activate Merchant
+    Merchant->>- SwedbankPay: GET <MobilePay payment>
+    activate  SwedbankPay
+    SwedbankPay-->>-Merchant: Payment response
+    activate Merchant
+    Merchant-->>-Merchant: Display payment Status
+```
```

Comment:
Note: This diagram might need a second opinion on (which is why this is a draft, and not normal Pull Request).

While the flow is close to how Vipps does the flow, there are some minor changes, mostly in the names of the sections.
In general, the merchant creates the purchase which we let them get the link to for their webstore. From there they load our instrument and then the payer clicks the button, the purchase is initialized towards MobilePay that gives us the URL to redirect the payer to. From here, they enter in their phone number (or prefilled by merchant) which then prompts the payer to check their app.

The MobilePay API contacts our API with the card data before we handle it, sending the result of the process back to the API. MobilePays own UI periodically checks their own API before redirecting the user back to the merchant, who then proceeds to call our API to check if the purchase was successful. Hopefully this comes through in the diagram, but it might need some tweeking.

Reply:
My confidence is a bit shaky on this one. I think we might have to check with the cavalry. @asbjornu , can you give the sequence diagram a stamp of approval?

Reply:
It would be easier to read if each `activate` and `deactivate` statement caused its contents to be indented. Alternatively, a screenshot would be very helpful.

Reply:
I provided a screenshot to you on Slack and I can indent the statements in the MD if needed, I just decided to follow the same setup as Vipps since the flow and code is closely similar, and I wanted to make sure this fork / pull request followed as closely as possible to your other documentation.

Reply:
![image](https://user-images.githubusercontent.com/22122794/98668820-b5d62c00-2350-11eb-93ba-86d9bb2ee57a.png)

Reply:
Please add `participant` with human readable aliases to the diagram. See `payment-order-other-features.md` for example:

https://github.com/SwedbankPay/developer.swedbankpay.com/blob/547571ae33d5d88c53e652367b690c5c37e87577/_includes/payment-order-other-features.md#L1014-L1017